src/dndbeyond/content-scripts/character.js

```diff
@@ -857,24 +857,97 @@ function displayAction(paneClass) {
     });
 }
 
+function handleCustomText(paneClass) {
+    // Function Objective: Search for and handle block of VTT Message text...
+    //...in pane Notes or Description (Priority descending in that order.)
+    const regexp = /```\[([^\]]*)\]\s*((\S|\s)*?)\s*```/g;
```

Comment:
The regexp would match `[]` on its own. Also, I don't see the need for the \`\`\` I think it's just confusing, if we use `[before-beyond20] as the mark, and anything on that line gets sent as a macro basically, that should suffice to make it clear.
I'd replace the regexp with this : 
`/^\[((?:before-beyond20)|(?:after-beyond20)|(?:replace-beyond20))\]\s*(.*)$/`
(did not test)
Note that with `/g`, you'll catch all occurences, and you're not handling multiple cases correctly.

Reply:
The goal of using the ` ``` ` delimiters was to allow users to send multi-line content to the VTT without needing to send many individual messages. 
Example multiline message: 
```
PlayerName attacks with flavour text!
AttackRoll Here
It does DamageRoll damage!
More optional flavour text and inline content here!
```
Using ` ``` ` as the delimiter was an arbitrary choice. In theory, it could be anything. (Although I avoided using `[]`, `{}`, and `()`, as Roll20 uses them in their macros, and I was trying to distinguish between Beyond20 syntax and Roll20 (and keep parsing simple)).

I do think that some sort of multi-line support would be a good idea though. What do you think?

Reply:
Good point about multiline, I hadn't thought of that. I don't like the \`\`\` though, it would be confusing to a lot of people I think who aren't used to markdown or aren't tech savvy (I expect half the users will complain that `'` doesn't work not realizing \` is a different character).
How about `[before] some test [/before]` ?